F-Clip Docker Image

kirill-ivanov-a opened the pull request:
Solved tasks:
- [x] F-Clip adapter code added;
- [x] Docker Image created.

The adapter code is in `run.py` and `adapter.py`.

kirill-ivanov-a committed:
F-Clip adapter added

- F-Clip adapted;
- Docker Image created.

kirill-ivanov-a committed:
F-Clip adapter added

- F-Clip adapted;
- Docker Image created.

kirill-ivanov-a committed:
Typo corrected

kirill-ivanov-a committed:
Typo corrected

DmiitriyJarosh commented on F-Clip/adapter.py:
```diff
@@ -0,0 +1,112 @@
+        batch_size: int = 2,
+    ):
+        self.image_path = image_path
+        self.lines_path = f"{output_path}/lines"
```
why are joins of path hardcoded?

DmiitriyJarosh commented on F-Clip/adapter.py:
```diff
@@ -0,0 +1,112 @@
+        makedirs(self.lines_path, exist_ok=True)
+        makedirs(self.scores_path, exist_ok=True)
+
+        if torch.cuda.is_available():
```
will docker work without cuda? Is it possible to launch eval without cuda?

DmiitriyJarosh commented on F-Clip/adapter.py:
```diff
@@ -0,0 +1,112 @@
+                for i, meta in enumerate(metadata):
+
+                    results = {}
+                    for k, v in heatmaps.items():
```
lets use better names than `k, v`

kirill-ivanov-a committed:
F-Clip added

- F-Clip original code added;
- README updated.

kirill-ivanov-a committed:
Delete Dockerfile

kirill-ivanov-a committed:
Merge branch 'f-clip' of github.com:prime-slam/line-detection-dockers into f-clip

kirill-ivanov-a committed:
CPU supported

- CPU support added;
- Hardcoded paths removed
- README updated;
- minor cleanup.
